pkg/gadgets/utils/test/doc.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The utils/test package implements a set of data types and helper
+// functions to test tracers without any external dependency.
+//
+// The main components are:
+// - Runner: A type that simulates a container. It's able to execute
+// golang functions in different namespaces, under different user ID,
+// etc.
+// - Test: A generic structure defining a test case.
+// - testRunner: A helper structure used to execute a series of tests.
+
+package test
```

Comment:
If this package is related to testing traces and it's only for testing, perhaps it would make sense to put it in `pkg/gadgets/trace/internal/test` so it does not become part of the module API and it's closer to the use?

Reply:
Yes, I had `pkg/gadgets/internal/test` in mind! Cool.